### DISCUSSION TITLE
Determining potential/gradients at specific point in box, which is not an atom

### DISCUSSION THREAD

**Discussion by jacekkozuch** [TinkerTools/tinker-gpu#222] [Q&A]
Hi,

I'm wondering if it is possible to determine the potential or gradients anywhere in space, but not on an atom. I tried to introduce a dummy atom, but defining MULTIPOLE parameters will induce dipoles into the environment and CHARGE does not interact at all with MULTIPOLE and POLARIZE parameters (if I haven't made any mistake trying this). Is there any other way to specify a "position" or a "probe site"?

Thanks in advance!

Best,
Jacek

**Comment by zhi-wang** [comment]
I'd assume you want to modify the code and to add some print statements in the source code. I'd do it in the Fortran code. A lot easier to start with.

If you add a dummy atom with multipole parameter

[no local frame] 0.0001
  0 0 0
  0
  0 0
  0 0 0

and set the polarizability to 0, I think the code will ensure the induced dipoles on the dummy atom to be 0. If you print out the result of subroutine DFIELD, it will give you accurate d/p damped permanent fields on the dummy atom. The perturbation of the field values (and induced dipoles if you are also interested in them) on other atoms due to the dummy atom will be minimal. I didn't set the dummy atom to complete zero multipole because I'm not sure what the fortran code will do -- it may ignore this atom because it may not be recognized as a multipole site. Tinker9 will keep the dummy atom as a multipole site in this situation though.

If you need the electrostatic potential calculated by 1X-mscale coefficients I don't think there is any routine readily available.

If the charge on the dummy atom is big, the error in the gradients will be big. If the charge is small, error in the gradients will be smaller but the numerical gradient of the dummy atom will be small too. So I dunno how useful the gradient will be.

**Reply by jacekkozuch** [reply]
Hi Zhi,
thanks a lot for the suggestion! 
In this case, I will try first if I get meaningful results if I place a dummy just with a 0.00001 monopole into my box, with a new atom type introduced in the key file. If this works well, then I'll consider looking at the code.
Best,
Jacek
